src/runtime_src/core/edge/drm/zocl/zocl_ioctl.c

```diff
@@ -562,3 +563,21 @@ zocl_read_axlf_ioctl(struct drm_device *dev, void *data, struct drm_file *filp)
 	vfree(axlf);
 	return ret;
 }
+
+int
+zocl_info_cu_ioctl(struct drm_device *dev, void *data, struct drm_file *filp)
+{
+	struct drm_zocl_info_cu *args = data;
+	struct drm_zocl_dev *zdev = dev->dev_private;
+	struct sched_exec_core *exec = zdev->exec;
+
+	if (!exec->configured) {
+		DRM_ERROR("Schduler is not configured\n");
+		return -EINVAL;
+	}
+ 
+	if(args->paddr != 0xFFFFFFFF) {
```

Comment:
Can you please explain why do you check args->paddr != 0xFFFFFFFF?

Reply:
actually not needed. 
removed